### ISSUE TITLE
Issue that occurred during `prisma db pull`

### ISSUE THREAD

**Issue by aslaker** [prisma/prisma#14647]
Hi Prisma Team! Prisma Migrate just crashed. 

## Command

`db pull`

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v16.15.1           | 
| Prisma CLI  | 4.1.1              | 
| Engine      | 8d8414deb360336e4698a65aa45a1fbaf1ce13d8| 

## Error
```
Error: [libs/datamodel/connectors/dml/src/model.rs:494:29] Hi there! We've been seeing this error in our error reporting backend, but cannot reproduce it in our own tests. The problem is that we have a primary key in the model `addressAttributesString` that uses the column `productId` which we for some reason don't have in our internal representation. If you see this, could you please file an issue to https://github.com/prisma/prisma so we can discuss about fixing this. -- Your friendly prisma developers. 
```

**Comment by aslaker** [comment]
Not sure if there is a workaround for this or not, but this was the template that prisma created for me.  Apologies if it is a duplicate of another issue.

**Comment by xMorthi** [comment]
same thing as #14636
run `prisma db pull` using version `4.0.0` then then after creating models upgrade to `4.1.1`

**Comment by tomhoule** [comment]
This should be fixed in Prisma 4.2, to be released tomorrow (https://github.com/prisma/prisma-engines/pull/3108)

**Comment by aslaker** [comment]
Excellent!  Thank you so much @tomhoule
